### PR TITLE
Beta-loading-skeleton

### PR DESCRIPTION
**Pull request by bjlagman12**
**Task Link:** https://synapsepay.monday.com/boards/180157847/pulses/371815143

**Description:**

LoadingSkeleton

Added the LoadingSkeleton component into the dev-ui-docz

The DocZ file displays a basic LoadingSkeleton with a specified height, width, and borderRadius

Props display all the necessary information the render the LoadingSkeleton

**Updates:**
none

**Notes:**
none

**Visuals:**
LS-basic

### CODE REVIEW

File: doczrc.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 export default {
   indexHtml: './index.html',
-  menu: ['Home', 'Button', 'Input','Display Card', 'Dropdown', 'CheckboxGroup', 'RadioGroup', 'Form', 'Modal', 'Panel', 'Table', 'Tag'],
+  menu: ['Home', 'Button','CheckboxGroup','Display Card', 'Dropdown','Form', 'Input', 'Loading Skeleton', 'Modal', 'Panel','RadioGroup', 'Table', 'Tag'],
```
Comment (nicklascschmidt) [inline doczrc.js]: Good call rearranging these-- maybe not a bad idea to break them into separate lines so it's easier to read through (also for Git). Somethin like this:
```
menu: [
    'Home', 
    'Button',
    'CheckboxGroup',
    'Display Card',
    'Dropdown',
    'Form',
    'Input',
    'Loading Skeleton',
    'Modal',
    'Panel',
    'RadioGroup',
    'Table',
    'Tag',
  ],
```

File: src/pages/loadingskeleton/loadingSkeleton.mdx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  height='100px'
+  width='840px'
+  borderRadius='10px'
+/>
+```
+## Props
+
+| Property                | Type                    | Required  | Description                                 |
+| :-------                | :---                    | :-------  | :----------                                 |
+| height                  | `strin`                 | false     | Height of LoadingSkeleton, defaults to 100%              |
```
[the remark this answers is not in the document]
Comment (bjlagman12) [reply src/pages/loadingskeleton/loadingSkeleton.mdx]: updated to string

File: src/pages/loadingskeleton/loadingSkeleton.mdx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+<LoadingSkeletonContainer />
+
+```jsx
+<LoadingSkeleton 
+  height='100px'
+  width='840px'
+  borderRadius='10px'
+/>
+```
```
[the remark this answers is not in the document]
Comment (bjlagman12) [reply src/pages/loadingskeleton/loadingSkeleton.mdx]: added another space

File: doczrc.js (hunk 1 above)
Comment (bjlagman12) [reply doczrc.js]: Fixed menu to make it easier to read

**Comment by bjlagman12** [comment]
Changes have been implemented

**Pull request merged**